### PR TITLE
Updated Error codes and created README

### PR DESCRIPTION
**Pull request by SidhaantAnand**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -37,6 +37,7 @@ app.post("/checkClaim", function (req, res) {
 });
 
 const server = app.listen(port, () => {
+  console.log('e');
```
Comment (dwertheimer) [inline server/index.js]: We should remove this console.log

File: server/routes/check_verification.js [hunk 2]
```diff
@@ -43,7 +48,7 @@ module.exports = function(req,res,constants,helpers,Sentry,client,base,airtableR
                     },
                     sentry_extras, Sentry
                   );
-                  res.status(400).send(reason1);
+                  res.status(500).send(reason1);
```
Comment (dwertheimer) [inline server/routes/check_verification.js:51]: 500 errors are usually server errors. I would suggest using something in the 400's

File: server/routes/check_verification.js [hunk 3]
```diff
@@ -58,20 +63,20 @@ module.exports = function(req,res,constants,helpers,Sentry,client,base,airtableR
                     },
                     sentry_extras, Sentry
                   );
-              res.status(400).send(reason2);
+              res.status(500).send(reason2);
```
Comment (dwertheimer) [inline server/routes/check_verification.js:66]: Same comment here

File: server/routes/check_verification.js [hunk 4]
```diff
@@ -82,7 +87,7 @@ module.exports = function(req,res,constants,helpers,Sentry,client,base,airtableR
             { name: "Verification Check Failed", message: `Verification check failed for ${send_phone_num}` },
             sentry_extras, Sentry
           );
-          res.status(400).send(reason);
+          res.status(500).send(reason);
```
Comment (dwertheimer) [inline server/routes/check_verification.js:90]: ...and here too (500 => 403 or something)

File: server/routes/send_verification.js [hunk 5]
```diff
@@ -88,12 +93,12 @@ module.exports = function(req,res,constants,helpers,Sentry,client,base,airtableR
...
                  email,
               {
-               name: "Twilio Verification send error",
+               name: "Verification create error",
                message: `Could not create verification for ${send_phone_num} `,
               },
               sentry_extras, Sentry
             );
-            res.status(400).send(reason);
+            res.status(500).send(reason);
```
Comment (dwertheimer) [inline server/routes/send_verification.js:101]: here also

File: server/routes/send_verification.js [hunk 6]
```diff
@@ -108,7 +113,7 @@ module.exports = function(req,res,constants,helpers,Sentry,client,base,airtableR
           },
           sentry_extras, Sentry
         );
-      res.status(400).send(reason);
+      res.status(500).send(reason);
```
Comment (dwertheimer) [inline server/routes/send_verification.js:116]: and here

File: server/README.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
+# Backend service for SpotifySocial
```
Comment (dwertheimer) [inline server/README.md]: A lot of great changes here, @SidhaantAnand. I like the README

Comment (dwertheimer) [inline server/README.md]: A lot of great changes here, @SidhaantAnand. I like the README. What's SpotifySocial?

File: server/routes/send_verification.js [hunk 8]
```diff
@@ -41,7 +46,7 @@ module.exports = function(req,res,constants,helpers,Sentry,client,base,airtableR
           },
           sentry_extras, Sentry
         );
-        res.status(400).send(value.error_message || value.status);
+        res.status(500).send(value.error_message || value.status);
         return;
```
Comment (dwertheimer) [inline server/routes/send_verification.js:50]: change this 500

File: server/index.js (hunk 1 above)
Comment (SidhaantAnand) [reply server/index.js]: Hey I removed it in a commit

File: server/routes/check_verification.js (hunk 2 above)
Comment (SidhaantAnand) [reply server/routes/check_verification.js:51]: I kept the errors which are due to failed information fetch from the APIs we depend on - Airtable, Google Place details and Twilio as 500

Comment (SidhaantAnand) [reply server/routes/check_verification.js:51]: Should I change to 400 something?

File: server/README.md (hunk 7 above)
Comment (SidhaantAnand) [reply server/README.md]: I edited this in a commit

File: server/routes/check_verification.js (hunk 2 above)
Comment (SidhaantAnand) [reply server/routes/check_verification.js:51]: On some research, 500s do not make sense in this case, I will change them all to 403(Forbidden)

**Pull request merged**